src/Voron/Data/Fixed/FixedSizeIterators.cs

```diff
@@ -164,6 +164,9 @@ public bool Skip(long count)
                 if (count != 0)
                     _pos += count;
 
+                if (_pos < 0)
```

Comment:
cc @arekpalinski maybe we need to move this to 4.2?

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-18079